lib/gameboard/render.rb

```diff
@@ -0,0 +1,121 @@
+module Gameboard
+
+  # to use the built in render class, pass Board#horizontal
+  # to Render::board
+  #
+  # All methods on Gameboard::Render are class methods
+  #
+  class Render
+
+    # ClassMethod: calls render methods in order of needed
+    #              execution.
+    #
+    # gameboard - A 2D array built by Board.horizontal
+    #
+    # Example
+    #
+    #    Render::board(Board.new(height:3, width: 3).horizontal)
+    #
+    def self.board(gameboard)
+      system("tput setab 0;") unless Gem.win_platform?
+      width = gameboard[0].length
+      spacing
+      gameboard.reverse.each_with_index do |row, i|
+        row_separator(width)
+        format_cells(row, i)
+      end
+      row_separator(width)
+      column_numbers(width)
+      spacing
+      system("tput sgr0;") unless Gem.win_platform?
+    end
+
+    # ClassMethod: Clears the terminal for clean output
+    #
+    #
+    # Example
+    #
+    #    Render::clear
+    #
+    def self.clear
+      system("cls") || system("tput reset") || system("clear") || puts("\e[H\e[2J")
```

Comment:
I actually did know that, however I read that system has in built escaping to keep users from gaining too much access